api/db/schema.sql

```diff
@@ -1,9 +1,12 @@
 DROP TABLE IF EXISTS users;
 CREATE TABLE users (
   user_id INTEGER PRIMARY KEY AUTO_INCREMENT,
-  username VARCHAR(255) NOT NULL,
-  email VARCHAR(255) NOT NULL,
-  password VARCHAR(255) NOT NULL
+  username VARCHAR
+(255) NOT NULL,
+  email VARCHAR
+(255) NOT NULL,
+  password VARCHAR
+(255) NOT NULL
```

Comment:
is this on purpose?

Reply:
It was auto formatted like this when I made changes to the file.

Reply:
Unless you have any objections I think the other style is more readable